Server/src/openbmp.cpp

```diff
@@ -86,6 +86,8 @@ void Usage(char *prog) {
     cout << "     -p <port>         BMP listening port (default is 5000)" << endl;
     cout << "     -b <MB>           BMP read buffer per router size in MB (default is 15), range is 2 - 128" << endl;
     cout << "     -hi <minutes>     Collector message heartbeat interval in minutes (default is 5 minutes)" << endl;
+    cout << "     -tx_max_bytes <bytes>    Maximum transmit message size (default is 1000000)" << endl;
```

Comment:
I'm going to merge this, but will remove this from cli.  going forward, let's only put new config options in the the config file.  This way the cli is simple.